tern/__main__.py

```diff
@@ -153,6 +158,18 @@ def main():
     parser_report.add_argument('-w', '--raw-image', metavar='FILE',
                                help="Raw container image that exists locally "
                                "in the form of a tar archive.")
+    parser_report.add_argument('-l', '--layer', metavar='LAYER_NUMBER',
+                               type=int, default=0,
+                               help="Layer number of the image to analyze."
+                               " Base OS layer is 1. Can only be used with"
+                               " -i --docker-image analysis. Default to 0"
+                               " which means the feature is turned off")
+    parser_report.add_argument('-li', '--layer-inclusive',
+                               metavar='LAYER_INCLUSIVE',
+                               type=bool, default=False,
+                               help="Usable only with -l --layer parameter."
+                               " When set to true, report will include all"
```

Comment:
What do you think of instead of having this variable be True/False we just check if it was invoked? i.e. if the user wants the inclusive option they can run `tern report -i <image> -layer 2 --layer_inclusive` instead of having to explicitly put true or false as the option?

Reply:
as in adding const=True action='store_const' ?

Reply:
Yes! Thank you for clarifying. What do you think? Does that make sense to you from a usability perspective?

Also, FYI I am still reviewing and testing the actual code changes so don't feel the need to push all these changes right now. Working as fast as I can though.

Reply:
Yeah, I'll adjust it that way, seems good!
No worries, I'm just finishing my work so I'll check the comments tomorrow. 
I wanted you to take a look at the change from pushing them through functions as args (as discussed in issues) to making them class attributes (as some func got too many args and linters complained)
Sorry that its still bit messy